src/cmds/show/user.ts

```diff
@@ -0,0 +1,32 @@
+import netrc from 'netrc'
+import { logFailure, log, logSuccess } from '../../logger'
+import { getNetrcEntryId, getSessionToken } from '../../utils/auth'
+import msg from '../../utils/msg'
+
+const CMD = 'user'
+
+function addUserCmd(cmd) {
+    cmd.command(CMD).action(showUser)
+}
+
+/**
+ * Show the current user.
+ */
+async function showUser() {
+    const { token, error } = getSessionToken()
```

Comment:
To avoid duplicate logic reading from `netrc` multiple times, let's just have a function inside of `utils/auth.ts` called `getSessionLogin`, that's almost identical to `getSessionToken` but just returns the `.login` instead of `.password`. Then this command's logic should be super simple with a `const { login, error } = getSessionLogin()`